api/v1alpha1/nificluster_types.go

```diff
@@ -281,7 +283,10 @@ type NodeConfig struct {
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
 	// Additionnal annotation to attach to the pod associated
 	// https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set
-	NodeAnnotations map[string]string `json:"nifiAnnotations,omitempty"`
```

Comment:
I discovered & fixed a bug where `NodeAnnotations` was [documented](https://orange-opensource.github.io/nifikop/docs/5_references/1_nifi_cluster/3_node_config#nodeconfig) as `nodeAnnotations` in the `NiFiCluster` CRD . However, it was actually `nifiAnnotations`. I've renamed it here (and it's reflected in the CRD) to match the documentation. This is technically an API breaking change, though.

Reply:
Created #151 to cover this issue